support_scripts/download_orthodb_proteinset.sh

```diff
@@ -166,6 +176,6 @@ grep -c -e ">" -I ${BASENAME}*.fa
 
 ## Print command to rename files:
 echo -e -n "\n## Renaming final output files:\n"
-perl ${PERL_SCRIPTS_DIR}/quick_rename.pl ${BASENAME}_final.out. ${CLADE_NAME}_orthodb_proteins. prefix
+perl ${PERL_SCRIPTS_DIR}/Quick_rename.pl ${BASENAME}_final.out. ${CLADE_NAME}_orth${ODB_VERSION}_proteins. prefix
```

Comment:
Did you rename this script? it's still lowercase in the repo https://github.com/Ensembl/ensembl-anno/blob/main/support_scripts_perl/quick_rename.pl

Reply:
No that was not my intent :D Good spotting thats Leanne ! Ill update now